scripts/run_integration_tests.sh

```diff
@@ -79,14 +79,16 @@ if [ ! -f "$VMLINUX_IMAGE" ]; then
 fi
 
 VIRTIOFSD="$WORKLOADS_DIR/virtiofsd"
+VUBRIDGE="$WORKLOADS_DIR/vubridge"
 QEMU_DIR="qemu_build"
 if [ ! -f "$VIRTIOFSD" ]; then
```

Comment:
```suggestion
if [ ! -f "$VIRTIOFSD" || ! -f "$VUBRIDGE" ]; then
```
We need to make sure to rebuild if one of the two is missing.